src/actinia_statistic_plugin/strds_sampling.py

```diff
@@ -86,186 +96,191 @@ class STRDSSampleResponseModel(ProcessingResponseModel):
                     "Number of centroids: 0",
                     "Number of areas: 0",
                     "Number of isles: 0",
-                    ""
+                    "",
                 ],
-                "stdout": ""
+                "stdout": "",
             },
             {
                 "executable": "t.rast.sample",
                 "parameter": [
                     "column=id",
                     "points=input_points",
                     "where=start_time >'2010-01-01'",
-                    "strds=temperature_mean_1950_2013_yearly_celsius@PERMANENT",
+                    "strds=temperature_mean_1950_2013_yearly_celsius@"
+                    "PERMANENT",
                     "output=/tmp/tmpgU9ITw",
                     "-rn",
                     "--o",
-                    "--v"
+                    "--v",
```

Comment:
There is often a `,` added at the end of a list, dict or aguments. Is that right/ok? (Just picked one example here)

Reply:
Yes.  Black adds these commas. It is okay and make no different in the code functionallity